partials/language-specific-deploy/php.md

```diff
@@ -480,7 +494,9 @@ Then, set `APP_LOG=syslog` as Clever application environment variable.
 
 ## Using HTTP authentication
 
-Using basic HTTP authentication, PHP usually gives you the user and password in variables named `$_SERVER['PHP_AUTH_USER']` and `$_SERVER['PHP_AUTH_PW']`. On Clever Cloud, we have enabled an option in Apache to pass directly the Authorization header even though we are using FastCGI; still, PHP does not use the header and the variables mentioned before are empty.
+Using basic HTTP authentication, PHP usually gives you the user and password in variables named `$_SERVER['PHP_AUTH_USER']` and `$_SERVER['PHP_AUTH_PW']`.
```

Comment:
```suggestion
Using basic HTTP authentication, PHP usually handles the values of user and password in variables named `$_SERVER['PHP_AUTH_USER']` and `$_SERVER['PHP_AUTH_PW']`.
```
the "you" is useless in this context
"user" and "password" are better defined as values, for clarity